### ISSUE TITLE
Tinker's tools sometimes or often available in starting Bonus Chest

### ISSUE THREAD

**Issue by lafflam** [JasonMcRay/Infitech-2-reloaded#3]
What happens:
Wooden Tinker's tools are sometimes or often available in the optional starting Bonus Chest, and they appear to be infinitely repairable in the player's 2x2 crafting grid.

What you expected to happen:
Since I assume Tinker's tools are supposed to be disabled in this modpack, I expect that they would not be available in the optional starting Bonus Chest.

(This also raises the question about whether they might show up in other loot chests. I have not found any chests yet, but will report back if I do.)

Steps to reproduce:
1. Single Player > Create New World > More World Options > Bonus Chest: ON
2. Create New World
3. Open the chest that spawns nearby and the player will sometimes or often find wooden Tinker's tools (axe, pick, etc.)

Modpack Version: Infitech 2 - Pre-loaded-0.0.1

I have not modified the pack in any way.

This issue is happening on the CLIENT in SINGLE PLAYER.
I have not tested this on server, but since there is no Bonus Chest on a server, I assume this issue is not applicable on a server.

Log: N/A

**Comment by JasonMcRay** [comment]
Loot chests is something I will need to definitely check. Completely forgot about this, since not many ppl uses Bonus Chests. Thanks

**Comment by lafflam** [comment]
I agree that not many people use Bonus Chests (I certainly don't), but I started looking into this problem anyway and I think I have found a fairly easy compromise / workaround. Simply change the following:

IguanaTinkerTweaks > main.cfg

  B:easyToolRepair=false

  I:repairsLimit=0

The player might still receive a few wooden Tinkers Pickaxes and Hatchets in the Bonus Chest, but when they break they can never be repaired.

Hope this helps for now.
